src/to_sql.rs

```diff
@@ -148,15 +149,20 @@ impl ToSQL for Expression {
                 exp.to_sql(out, conf)?;
                 write!(out, ")")
             }
-            Expression::BinaryOperator(l, op, r) => {
+            Expression::BinaryOperator(l, op, r, p) => {
                 //TODO(justin): order of operations is preserved by parenthesizing
```

Comment:
You should remove/update this comment since we're not always injecting parentheses.